tests/functional/workloads/cnv/test_cnv_device_replacement.py

```
@@ -36,15 +36,16 @@ def setup(self, request, project_factory, multi_cnv_workload):
-        # Create a project
+        logger.test_step("Create project and deploy CNV workload VMs")
         proj_obj = project_factory()
         (
             self.vm_objs_def,
             self.vm_objs_aggr,
             self.sc_obj_def_compr,
             self.sc_obj_aggressive,
         ) = multi_cnv_workload(namespace=proj_obj.namespace)
-        logger.info("All vms created successfully")
+        all_vms = self.vm_objs_def + self.vm_objs_aggr
+        logger.info(f"Created {len(all_vms)} VMs successfully")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)